c10/core/TensorOptions.h

```diff
@@ -558,6 +568,8 @@ inline DeviceType computeDeviceType(TensorTypeId tid) {
     return DeviceType::CUDA;
   } else if (tid == SparseHIPTensorId()) {
     return DeviceType::HIP;
+  } else if (tid == MkldnnCPUTensorId()) {
```

Comment:
wait, why do we have MKLDNNTensorId above?  What's the plan here?

Reply:
@gchanan MKLDNNTensorId() should belong to Caffe2 backend and was created when we integrated MKL-ML library into Caffe2. It was named "MKLDNN" by accident. It is obsolete and should be removed. We will clean it up in later PR.